KMaina/Fast-Food-App
# #160292858 Page to View History of All Orders Made

#### What does this PR do?
Allows a user to view all the orders placed.
#### Description of Task to be completed?
The user should be able to see all orders placed.
#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`
 
- run `git checkout -b ft-user-order-history-160292858`
 
- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-user-order-history-160292858`
 
- In the folder where you cloned the repo, you should open the UI folder.
 
- Open the user-order-history.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160292858
